operations/workplace/people/people-policies.md

```diff
@@ -44,6 +44,12 @@ To enable Managers and Supervisors with these responsibilities, Mattermost provi
 
 Mattermost's Affirmative Action Plan can be found [here](https://docs.google.com/document/d/15PVfR5uI7TaWuSznLr3hmQd_4aAE-OR3AixVPEO-QF4/edit#heading=h.cbr53qvhhkxh).
 
+## Job abandonment
+
+When a staff member is absent from work for 3 consecutive workdays, and there is no entry on the [Time Away Calendar](https://www.google.com/url?q=http://app.rippling.com/api/feed/calendar/pto/company/are2zz48b0jkynqd/b91837056f386ea9355b4e71f456c24a381efea19840116f0779e81f053199e4/calendar.ics?company%3D658390ff2cae6bf84a79bcb5&sa=D&source=docs&ust=1718123593252322&usg=AOvVaw0l7BJEx680vP_7GJ520jMs), or a post in the [Stand-Up Channel](https://hub.mattermost.com/private-core/channels/stand-up) noting the time off, and the staff member fails to contact their manager or HR, they can be terminated for job abandonment unless otherwise required by local law.
```

Comment:
```suggestion
When a staff member is absent from work for 3 consecutive workdays, and there is no entry on the [Time Away Calendar](http://app.rippling.com/api/feed/calendar/pto/company/are2zz48b0jkynqd/b91837056f386ea9355b4e71f456c24a381efea19840116f0779e81f053199e4/calendar.ics?company%3D658390ff2cae6bf84a79bcb5&sa=D&source=docs&ust=1718123593252322&usg=AOvVaw0l7BJEx680vP_7GJ520jMs), or a post in the [Stand-Up Channel](https://hub.mattermost.com/private-core/channels/stand-up) noting the time off, and the staff member fails to contact their manager or HR, they can be terminated for job abandonment unless otherwise required by local law.
```